ceagle/main.py

```diff
@@ -44,6 +45,11 @@ def not_found(error):
     return flask.jsonify({"error": "Not Found"}), 404
 
 
+@app.errorhandler(client.UnknownService)
+def handle_unknown_service(ex):
+    return flask.jsonify({"error": str(ex)}), 400
```

Comment:
I'd argue 404 is the correct code. It means client is not configured and all the respective urls are unavailable, while 400 indicates problems with request itself https://httpstatuses.com/400